### DISCUSSION TITLE
idle-iktah-tips-faqs-and-endgame-guides/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#16] [Comments]
# idle-iktah-tips-faqs-and-endgame-guides/

I have been playing Idle Iktah almost obsessively for the last 6 weeks, and have finally reached all 99s and almost 100% completion! In this post, I’ll answer common questions from the Discord, reveal plenty of gameplay tips, and include a small collection of mini-guides. Enjoy!

https://jakelee.co.uk/idle-iktah-tips-faqs-and-endgame-guides/

**Comment by 0sagittarius** [comment]
Hi!! Is the black pearl useful in the game? I've accidentally got one and have no idea what it is, but it's worth a lot of rare shells, should I sell it or keep it? Thank you so much for reading this!

**Reply by JakeSteam** [reply]
There's no purpose to it, besides being extremely rare :) I'd recommend selling it, or you could keep it as a kind of trophy.

**Reply by ShadeX** [reply]
Black pearls can be used in the tidekeeper. However the benefit likely isn't worth grinding for 3 black pearls.

**Comment by Alysraz** [comment]
Hello, the item "Rose Tinted Goggles" just can see the hide mob and show "?" in the "The Deep Forest" battle swap choice maps.
After show the hide mobs icon "?" then ues poison "Headhunter Potion" maybe can help the "Sasquatch" soon.

**Comment by okaw360** [comment]
You need to correct 3 black pearl for new full power tidekeeper

**Reply by JakeSteam** [reply]
Thanks, added info!

**Comment by imlps** [comment]
Thank you for the guide! 😍
